src/input/internal.tsx

```diff
@@ -199,6 +202,7 @@ function InternalInput(
       {...(__injectAnalyticsComponentMetadata
         ? getAnalyticsMetadataAttribute({ component: componentAnalyticsMetadata })
         : {})}
+      {...copyAnalyticsMetadataAttribute(rest)}
```

Comment:
Is this fine that `copyAnalyticsMetadataAttribute` ignores the `__injectAnalyticsComponentMetadata` flag and renders the metadata attribute regardless?